components/ContentTemplates/FormsTemplate.tsx

```diff
@@ -1,8 +1,14 @@
+import dynamic from "next/dynamic"
 import { formsPageNavigation } from "../../data/pageNavigationLists"
 import { CodeBlock } from "../CodeBlock/CodeBlock"
 import { NavPage } from "../NavPage/NavPage"
 import { PageUpdated } from "../PageUpdated/PageUpdated"
 import { TemplateSection } from "../TemplateSection/TemplateSection"
+import { formatCss, formatHtml } from "../../utils"
+
+const Playground = dynamic(() => import("../Playground/Playground"), {
```

Comment:
Is there any reason this needs to be dynamic? Why can't it be server side rendered like the rest of the content on the page?

Reply:
By default, sandpack provides client-side rendering. When not imported dynamically like above where I set ssr: false, it was causing Hydration problem. To avoid it, I used dynamic rendering as a first step. But it's possible to server render . More [on it](https://sandpack.codesandbox.io/docs/guides/ssr)

I need your opinion here, whether I should use server side rendering here or go ahead with default client side?